django_app_server_db_server/deployment/roles/app_server/tasks/main.yml

```diff
@@ -114,14 +127,21 @@
   # - name: Checkout eclaim source code
   # git: repo={{ deployment_url }} dest=/opt/eclaim_revamp force=yes
 
-- name: Setup pip requirements
+- name: Downgrade pip to 6.1.1
+  shell: pip install pip==6.1.1
+  when: app_version == "eclaim"
+
+- name: Setup pip requirements for eclaim_revamp
   pip: requirements=/opt/eclaim_revamp/{{ item }}.txt virtualenv=/opt/miniconda2
   with_items:
      - requirements
      - req_test
-  # - name: Setup pip requirements
-  # pip: requirements=/opt/eclaim_revamp/req_test.txt
-  #
+  when: app_version == "eclaim_revamp"
+
+- name: Setup pip requirements for eclaim
+  pip: requirements=/opt/eclaim/eclaim/requirements.txt virtualenv=/opt/miniconda2
+  when: app_version == "eclaim"
```

Comment:
Probably here you can use the trick above for the `mercurial` one to add a conditional to your `with_items` rather than adding a whole section like this.